src/safari-app-extension/NSCustomPersistantContainer.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NSCustomPersistantContainer.swift
+//  AlpheiosSafariExtension
+//
+//  Created by Kirill Latyshev on 19/08/2019.
+//  Copyright © 2019 Alpheios Team. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class NSCustomPersistentContainer: NSPersistentContainer {
+    
+    override open class func defaultDirectoryURL() -> URL {
+        var storeURL = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "W79WNX2NKZ.AlpheiosExtensionSharedData")
```

Comment:
This is an AppGroup name from the Capabilities tab in project settings.
![image](https://user-images.githubusercontent.com/18631055/64595400-efb14880-d3c2-11e9-8031-de92ae82f396.png)
The first part of it is a team prefix that is the one matching the "The Alpheios Project Ltd." team. The second part is that some name I come out with (it can be anything and we can change it if we need to).

In order to share a common core data both a containing app and an app extension must be in the same group. A certain folder will be allocated for the group in a local file system. All group members will have a shared access to this folder. The line of code mentioned is getting a URL of that shared folder in order to store an underlying SQLite DB files with the data in there.

Reply:
ok. I just realized today that the W79... is our team id, when I renewed our app store membership.